pom.xml

```diff
@@ -69,6 +69,16 @@
             <timezone>-6</timezone>
         </developer>
     </developers>
+    <!-- modules shared among profiles -->
+    <modules>
```

Comment:
so we had common at one point then I think it was the spark 2.x support for explain that forced us to move within each profile.  That is gone so this should be good to put back.